src/services/userService.js

```diff
@@ -14,25 +14,55 @@ const getUserSignUp = async (
   gender,
   password,
   profileImg,
-  age
+  age,
+  admin
 ) => {
   validateId(userId);
   validateNickname(nickname);
   validatePassword(password);
 
   const saltRounds = 10;
   const hashedPw = await bcrypt.hash(password, saltRounds);
-
+  
   return await userDao.createUser(
     userId,
     nickname,
     gender,
     hashedPw,
     profileImg,
-    age
+    age,
+    admin
   );
 };
 
+const signIn = async ( userId, password ) => {
+  const user = await userDao.signIn(userId);
+  const checkHash = async (password, hashedPassword) => {
+      return await bcrypt.compare(password, hashedPassword);
+  }
+
+  const checkPassword = await checkHash(password, user.password);
+  if(!checkPassword) {
+      const err = new Error(`유저 정보가 일치하지 않습니다.`);
+      err.statusCode = 400; 
+      throw err;
+  }
+    const JWT_KEY = process.env.KEY;
+    
+  const jwtToken = jwt.sign({id:user.id, user_id:user.user_id, admin:user.admin}, JWT_KEY);
```

Comment:
한 줄이라도 더 줄여볼 수 있겠습니다.
```suggestion
  const jwtToken = jwt.sign({id:user.id, user_id:user.user_id, admin:user.admin}, process.env.KEY);
```